nemo/collections/tts/torch/data.py

```diff
@@ -53,15 +54,6 @@
 from nemo.core.classes import Dataset
 from nemo.utils import logging
 
-EPSILON = 1e-9
```

Comment:
Looks like your merge is trying to revert 'RADTTS bits' PR (and a fix to it) we merged in the last cycle. 
My guess is version in main should work as is  - if not, take a look at the diff and very carefully cherry-pick isolated changes what may have been missing.